src/lib/tokenPrices.ts

```diff
@@ -5,7 +5,9 @@ import { Maybe } from './types';
 
 const tokenToUsdCache: { [token: string]: number | undefined } = {};
 
-export const tokenToUsd = async (token: string): Promise<number> => {
+export const tokenToUsd = async (
+	token: string,
+): Promise<number | undefined> => {
```

Comment:
I don't like how this can potentially return `undefined`
If this function fails for some reason, it should throw an error, not return `undefined`.

Reply:
Understood, lets say that fails to retrieve data, should console the error and not display the value? ( $0 in that case )